sigh/lib/assets/resign.sh

```diff
@@ -721,11 +707,9 @@ function resign {
         done
 
         # Replace old bundle ID with new bundle ID in patched entitlements
-        # Read old bundle ID from the old Info.plist which was saved for this purpose
-        OLD_BUNDLE_ID="$(PlistBuddy -c "Print :CFBundleIdentifier" "$TEMP_DIR/oldInfo.plist")"
```

Comment:
any idea why this was done this way? this PR now just uses `CURRENT_BUNDLE_IDENTIFIER` further down instead of this "copy" - is this safe?